app.js

```diff
@@ -3,14 +3,21 @@ const speakersCountry = document.querySelectorAll('.speakers-country');
 const speakersDescription = document.querySelectorAll('.speakers-description');
 const speakersBgImage = document.querySelectorAll('.img-transparent-bkgrnd');
 const speakersImage = document.querySelectorAll('.speakers-images');
-const morebtn = document.querySelector('#moreBtn');
-const speakercontainerfordesktop = document.querySelectorAll('.speakers-info-container ul');
-
-morebtn.addEventListener('click', () => {
-  speakercontainerfordesktop.forEach((item) => {
-    item.classList.remove('display-none');
-  });
+const dh = document.getElementById('dh');
+const more = document.getElementById('more');
+const button = document.getElementById('moreBtn');
+let isMoreVisible = false;
+button.addEventListener('click', () => {
+  isMoreVisible = !isMoreVisible;
+  if (isMoreVisible) {
+    more.style.display = 'block';
+    button.innerText = 'Show less';
+  } else {
+    more.style.display = 'none';
+    button.innerText = 'Show More';
+  }
 });
+
 const speakersInfo = [
```

Comment:
Great design @godybuoy 🤝

- [ ] Kindly ensure your speakers' section is well aligned so that your design may replicate the design provided.
- Your design
![Your feature-speakers](https://user-images.githubusercontent.com/63917725/235324949-a004ef8d-fab1-4be9-beac-436c6c2e2690.PNG)
- Given design
![Given-feature-speakers](https://user-images.githubusercontent.com/63917725/235324956-d1821611-21cb-42fe-bbd5-ec28c9d599ec.PNG)